### PR TITLE
Core 8351 

### PR DESCRIPTION
**Pull request by sri2k1us**
- Replace Feedback icon with help icon
- Remove forum icon
- Add FAQ's link, Forum and Feedback link under help icon menu.
- Refactor variable names.
- Add help button to Data, Apps and Analyses and link to FAQs page.

### CODE REVIEW

File: de-lib/src/main/java/org/iplantc/de/desktop/client/views/DesktopViewImpl.java [hunk 1]
```diff
@@ -53,22 +53,24 @@
...
-    @UiField IPlantAnchor collaboratorsBtn;
-    @UiField IPlantAnchor systemMsgsBtn;
-    @UiField IPlantAnchor documentationBtn;
+    @UiField IPlantAnchor preferencesLink;
+    @UiField IPlantAnchor collaboratorsLink;
+    @UiField IPlantAnchor systemMsgsLink;
+    @UiField IPlantAnchor documentationLink;
     @UiField IPlantAnchor introBtn;
-    @UiField IPlantAnchor contactSupportBtn;
+ //   @UiField IPlantAnchor contactSupportBtn;
```
Comment (ash3rz) [inline de-lib/src/main/java/org/iplantc/de/desktop/client/views/DesktopViewImpl.java]: There are a few things in this file that are commented out that we could probably just go ahead and delete to keep things clean.

File: de-lib/src/main/java/org/iplantc/de/desktop/client/views/DesktopViewImpl.ui.xml [hunk 2]
```diff
@@ -31,35 +35,45 @@
...
-                            <menu:SeparatorMenuItem />
-                            <anchors:IPlantAnchor ui:field="documentationBtn"
+                            <menu:SeparatorMenuItem/>
+                            <anchors:IPlantAnchor ui:field="documentationLink"
                                                   text="{appearance.documentation}"/>
                             <anchors:IPlantAnchor ui:field="introBtn"
                                                   text="{appearance.introduction}"/>
-                            <anchors:IPlantAnchor ui:field="contactSupportBtn"
-                                                  text="{appearance.contactSupport}"/>
+                         <!--<anchors:IPlantAnchor ui:field="contactSupportBtn"
```
Comment (ash3rz) [inline de-lib/src/main/java/org/iplantc/de/desktop/client/views/DesktopViewImpl.ui.xml]: Same here - I'd favor deleting code over commenting it out.

File: de-lib/src/main/java/org/iplantc/de/desktop/client/views/DesktopViewImpl.java (hunk 1 above)
Comment (sri2k1us) [inline de-lib/src/main/java/org/iplantc/de/desktop/client/views/DesktopViewImpl.java]: Ahh I just forgot to get rid of them. I started with commenting out stuff that wasn't needed.

File: de-lib/src/main/java/org/iplantc/de/desktop/client/views/DesktopViewImpl.ui.xml (hunk 2 above)
Comment (sri2k1us) [inline de-lib/src/main/java/org/iplantc/de/desktop/client/views/DesktopViewImpl.ui.xml]: Same here!

**Pull request merged**